mltsp/tests/test_build_model.py

```diff
@@ -3,13 +3,14 @@
 import numpy.testing as npt
 import os
 import pandas as pd
+import numpy as np
 from sklearn.externals import joblib
 import shutil
 
 def test_csv_parser():
     """Test CSV file parsing."""
     colnames, data_rows = build_model.read_data_from_csv_file(
-        os.path.join(os.path.dirname(__file__), "Data/csv_test_data.csv"))
+        os.path.join(os.path.dirname(__file__), "data/csv_test_data.csv"))
```

Comment:
Sorry for this mess; in my casperjs branch there are patches to make it easier next time.